x-pack/elastic-agent/pkg/agent/program/testdata/single_config-heartbeat.yml

```diff
@@ -0,0 +1,18 @@
+inputs:
+- type: synthetics/http
+  id: unique-http-id
+  name: my-http
+  schedule: '*/5 * * * * * *'
+  hosts: ["http://localhost:80/service/status"]
```

Comment:
The one change I'd like to make vs. heartbeat as it is today is to make this singular instead of plural.

Reply:
@blakerouse If i understand it correctly, the files you have here are "only" for testing but in the end we just forward the blog.

Reply:
@ruflin Correct, it's just for testing. We forward all the content to heartbeat, it could have an invalid name in there like `foo: bar` and that would also be sent.

I will still update the test so it matches the expected structure.